pkg/kubernetes/impersonate_roundtripper.go

```diff
@@ -8,7 +8,7 @@ type impersonateRoundTripper struct {
 
 func (irt *impersonateRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 	// TODO: Solution won't work with discoveryclient which uses context.TODO() instead of the passed-in context
-	if v, ok := req.Context().Value(AuthorizationHeader).(string); ok {
+	if v, ok := req.Context().Value(OAuthAuthorizationHeader).(string); ok {
```

Comment:
Just a note that this is dead code at the moment.

Reply:
I was planning to ask you this. Why don't we remove it for clarity?